packages/py3_typogrify.rb

```diff
@@ -7,6 +7,7 @@ class Py3_typogrify < Pip
   license 'BSD'
   compatibility 'all'
   source_url 'SKIP'
+  binary_compression 'tar.zst'
```

Comment:
This package still has `no_compile_needed`, so a binary compression value here is spurious.